### ISSUE TITLE
Problem : Can't view query

### ISSUE THREAD

**Issue by Yoshi-OOF** [WilliamVenner/SQLWorkbench#6]
Hey!
I have a problem I can't view any query and I don't know why :
Here is a screenshot :
http://prntscr.com/uocmqp

But utime save in the sv.db : 
http://prntscr.com/uocmup

But when i go in db browser that's work fine : 
http://prntscr.com/uocp1u

Can you help me? 
I say you thank you in advance.

**Comment by WilliamVenner** [comment]
Does it work if you use "Basic Mode"?

**Comment by Yoshi-OOF** [comment]
Wait i will try

**Comment by Yoshi-OOF** [comment]
Nope : 
![image](https://user-images.githubusercontent.com/18462508/94346292-dad64000-002b-11eb-99c8-c4689943772a.png)

![image](https://user-images.githubusercontent.com/18462508/94346301-e6c20200-002b-11eb-9d9b-11ef965f6605.png)

**Comment by MenschlichTv** [comment]
Every time I use the SQLWorkbench, I get the same errors... therefore I just write the select statements by myself...

**Comment by WilliamVenner** [comment]
No clue, everything works fine for me.

**Comment by RobertEikeland** [comment]
Yeah, I am also experiencing this issue. Clicking any of the tables throws up this error. (This is in SQLite)

**Comment by WilliamVenner** [comment]
I tested with both SQLite and MySQL D:

**Comment by MenschlichTv** [comment]
I've only used the workbench with SQLite tho...
But I have tried this in local multiplayer and also on multiple servers, and it didn't work on any of them...
Not only for me but also for all other admins.

**Comment by WilliamVenner** [comment]
I think it might have something to do with the compression of the queries, as net.WriteData/ReadData were changed recently and it's become clear that the number of "bytes" I'm inputting to the functions is grossly incorrect.

@MenschlichTv Do you think you could fiddle around with the code to confirm?

**Comment by WilliamVenner** [comment]
Saying that, I have no idea why it works for me in that case.

**Comment by MenschlichTv** [comment]
I also just saw in the screenshots that there should be an auto complete? Because I also never saw that ingame...
The only difference between IDE and basic mode was the coloring of keywords.. at least everytime I used it it was like that

**Comment by WilliamVenner** [comment]
You on x86_64? That also works for me.

**Comment by MenschlichTv** [comment]
Im playing with the default gmod version

**Comment by MenschlichTv** [comment]
just noticed that I also get this error:
[HTML] :1: Uncaught TypeError: Property 'SetSQLQuery' of object #<error> is not a function

**Comment by WilliamVenner** [comment]
You need to be on x86_64 branch, the default gmod web view stuff is too old to work properly with this.

**Comment by MenschlichTv** [comment]
Yup, works now.

**Comment by Craftgeeking** [comment]
Hello there ! , sorry to revive this issue but the same thing that happened to Yoshi happened to me : SQlite viewing doesn't work

So i was wondering if i needed another module like tmysql or mysqloo to view the **sqlite database ** ?
